git-commit-signing/README.md

```diff
@@ -0,0 +1,65 @@
+---
+display_name: Git commit signing
+description: Configures Git to sign commits using your Coder SSH key
+icon:  ../.icons/git.svg
+maintainer_github: phorcys420
+verified: false
+tags: [helper, git]
+---
+
+# git-commit-signing
+
+<!-- Describes what this module does -->
+
+```hcl
+module "git-commit-signing" {
+    source = "https://registry.coder.com/modules/git-commit-signing"
+}
```

Comment:
```suggestion
module "git-commit-signing" {
    source   = "https://registry.coder.com/modules/git-commit-signing"
    agent_id = "coder_agent.example.id" 
}
```
`agent_id` is a required variable for modules using `coder_script`

Reply:
forgot to push